docs/DeveloperGuide.md

```diff
@@ -257,42 +257,83 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | SoC Student        | find a module and the specifics of the module      | not spend too much time finding information on various websites |
+| `* * *`  | SoC Student        | add the deadline of my assignments for each module | easily keep track of my deadlines                               |
+| `* * *`  | SoC Student        | find a contact easily                              | not spend too much time looking for contacts                    |
+| `* *`    | SoC Student        | check my schedule for the day                      | plan ahead                                                      |
+| `* *`    | SoC Student        | organise consultations easily                      | not need to search various websites to organise consultations   |
+| `* *`    | SoC Student        | label my contacts                                  | keep track of my project groups                                 |
+| `* *`    | SoC Student        | organise the deadlines of my assignments           | see which deadline is most pressing                             |
+| `* *`    | SoC Student        | see the dates of my exams                          | better prepare for them                                         |
+| `*`      | SoC Student        | add miscellaneous events                           | better plan my time                                             |
+| `*`      | SoC Student        | archive the current information                    | reset for the new semester                                      |
+| `*`      | SoC Student        | keep track of my interview dates                   | make ample preparations                                         |
+| `*`      | SoC Student        | keep track of my weekly meetings                   | make preparations for them                                      |
+| `*`      | Teaching Assistant | access document links for all my slides            | share with the class I am teaching                              |
+| `*`      | Teaching Assistant | manage my student's consultation slots             | easily find the timing for their consultation                   |
+| `*`      | Teaching Assistant | access my module website                           | grade my student's submission                                   |
+| `*`      | Teaching Assistant | easily group my students' contacts together        | easily find them at once                                        |
 
-*{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `SoCompiler` and the **Actor** is the `user`, unless specified otherwise)
+
+**Use case:UC1 Add a person**
+
+**MSS**
+
+1. User requests to add person
+2. SoCompiler adds the person to the list of persons
```

Comment:
um, I think this is missing a few steps,
1. User request.......
2. System request for person details
3. User inputs details
4. SoC.....